analyze.py

```diff
@@ -382,6 +382,36 @@ def analyze_timeline(user_id, list_id, api, cache):
     users = fetch_users(timeline_ids, api, cache)
     return analyze_users(users, ids_fetched=len(timeline_ids))
 
+def analyze_my_timeline(user_id, api, cache):
+    # Timeline-functions are limited to 200 statuses
+    statuses = api.GetUserTimeline(screen_name=user_id, count=200, include_rts=True, trim_user=False, exclude_replies=False)
+    maxt = 0
+    for i in range(1, 10): #end of range is exclusive so this only goes to 9, so we're looking at max 2,000 tweets
+        if not maxt == statuses[-1].id - 1: #to prevent excessive API calls if already exhausted
```

Comment:
Here you could use a `break` statement to exit the loop more legibly.

Reply:
Thanks for the suggestion! Pushed an edit to do that.